sentry-spring/src/main/java/io/sentry/spring/graphql/SentrySpringSubscriptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package io.sentry.spring.graphql;
+
+import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import io.sentry.IHub;
+import io.sentry.graphql.ExceptionReporter;
+import io.sentry.graphql.SentrySubscriptionHandler;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.graphql.execution.SubscriptionPublisherException;
+import reactor.core.publisher.Flux;
+
+public final class SentrySpringSubscriptionHandler implements SentrySubscriptionHandler {
+
+  @Override
+  public Object onSubscriptionResult(
+      final @NotNull Object result,
+      final @NotNull IHub hub,
+      final @NotNull ExceptionReporter exceptionReporter,
+      final @NotNull InstrumentationFieldFetchParameters parameters) {
+    if (result instanceof Flux) {
+      Flux<?> flux = (Flux<?>) result;
+      return flux.doOnError(
+          throwable -> {
+            ExceptionReporter.ExceptionDetails exceptionDetails =
+                new ExceptionReporter.ExceptionDetails(hub, parameters.getEnvironment());
+            if (throwable instanceof SubscriptionPublisherException
```

Comment:
Do we want to unwrap this? Seems to always be there for Spring

Reply:
What do you mean?

Reply:
Here I'm unwrapping `SubscriptionPublisherException` to get its cause as this wrapper seems to always be there for Spring. I assumed it'd only mess up UX and grouping in Sentry so I unwrapped it.